### DISCUSSION TITLE
#44 - Stripes

### DISCUSSION THREAD

**Discussion by rafehimself** [cssbattle/forum#126] [Q&A]
where is the difference ? i don't see it 👀 
![image](https://user-images.githubusercontent.com/58576132/136675485-4c6dc9f8-3f0b-49da-8b43-644a56ebc509.png)
`<p><style>*{background:var(--,#1A4341)}*>*{--:#F3AC3C;height:20;margin:60;box-shadow:0 20px#1A4341,0 40px#F3AC3C,0 60px#1A4341,0 80px#F3AC3C,0 100px#1A4341,0 120px#F3AC3C,0 140px#1A4341,0 160px#F3AC3C}
  p {
    width: 300px;
    height: 300px;
    border-radius:50%;
    --:#1A4341;
    box-shadow:none;position:fixed;left:-210;top:-60;
    -webkit-box-reflect:right 100px}`

I even switched to pixels px from q that usually works despite being slithy less accurate, still not 100%

**Comment by chinchang** [comment]
Did you try our all new "see differences" option?
https://twitter.com/css_battle/status/1446908015756541954?t=vVi-THSgY2mH9dFpFiySqQ&s=19

**Reply by rafehimself** [reply]
I can see it now, something is wrong with stripes edges,
but that's beyond my level of understanding of CSS in order to fix it

**Comment by SelenIT** [comment]
The problem with getting 100% match at this target seems to become recurrent (e.g. #12, #26, #43). And I agree that it seems quite illogical that a perfectly placed and perfectly shaped circle is considered mismatch only because it's built with CSS properties that the backend "doesn't like", while it accepts the gradient-based solution looking much less perfect to the human eye. Maybe it can be fixed/tweaked somehow?.. 🥺

**Comment by rafehimself** [comment]
well ... the judging algorithm isn't perfect (there is still hope for humans 🤣 )
![image](https://user-images.githubusercontent.com/58576132/136723812-d3970426-cfe2-46fc-8750-ade624837cd1.png)
first one is a mismatch, while the second pass as 100% despite the ball being slightly displaced in supernova #83
